shared/naturalcrit/codeEditor/helpers/widget-elements/constants.js

```diff
@@ -5,26 +5,27 @@ export const HINT_TYPE = {
 	NUMBER_SUFFIX : 1
 };
 
-export const WIDGET_TYPE = {
-	SNIPPET        : 0,
+export const SNIPPET_TYPE = {
+	DEFAULT        : 0,
 	INLINE_SNIPPET : 1,
 	IMAGE          : 2,
```

Comment:
Can we name these "BLOCK", "INLINE", AND "INJECTOR"?

Reply:
Done!